client/src/Login.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import axios from 'axios';
+import {
+    BrowserRouter,
+    Switch,
+    Route,
+    Link
+  } from "react-router-dom";
+
```

Comment:
[nit] There's a bunch of string inconsistencies (" vs ') - again a linter will fix this so no big deal now